packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -5,7 +5,13 @@ import { screenReaderContainerStyles } from '../../../../lib/accessibility/Style
 import { pxToRem } from '../../../../lib'
 
 const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageVariables> = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
+  root: ({
+    props: p,
+    variables: v,
+    theme: {
+      siteVariables: { orange04, red },
```

Comment:
Can we move this to `chatMessageVariables`?

Reply:
Yep, will add mentionColor and importantColor variables there.

Reply:
totally agree - just to express my following thoughts on that, would limit it to two corresponding variables for now, `isImportantColor` and `hasMentionColor`, even given that each is applied in two different places. This will better reflect the fact that the same color is meant to be applied